code.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "name": "regional-hydrologic-forcings-ml",
+    "organization": "U.S. Geological Survey",
+    "description": "Repo for machine learning models for regional prediction of hydrologic forcing functions. FY22 regions: Delaware River Basin (DRB) region, and Upper Colorado River Basin (UCOL) region.",
```

Comment:
@jds485 Should this description be updated? I'm guessing it was auto-generated when we first created this repo, and now the regions listed are out of date.

Reply:
Good point - we can update this outside of this PR and I think this file will also be updated

Reply:
Well, I just updated the repo description on GitHub and this did not change, so I guess we can manually edit the description.